domain.go

```diff
@@ -208,6 +208,24 @@ func (d *Domain) PCIDevices() ([]PCIDevice, error) {
 	return devices, nil
 }
 
+// CPU returns a domain's CPU(s).
+func (d *Domain) CPU() ([]CPU, error) {
```

Comment:
One more quick thing, sorry!

Name this `CPUs` to match the other method names that return a slice of some type.